packages/components/menu/src/menu-item.scss

```diff
@@ -1,98 +1,110 @@
 :host {
-  --_background: var(--sl-color-select-item-background);
-  --_border-radius: var(--sl-border-radius-popover-default);
-  --_cursor: pointer;
-  --_focus-outline: var(--sl-color-focusring-default) solid var(--sl-border-width-focusring-default);
-  --_gap: 0.25rem;
-  --_indent: calc(var(--sl-menu-item-indent, 0) * 26px + 6px);
-  --_gap-shortcut: 0.5rem;
-  --_gap-sub-menu-indicator: 1rem;
-  --_padding: 6px;
-  --_submenu-offset: -5px;
-
```

Comment:
We are not setting font-size anymore? It inherits from the body, yes?

Reply:
That's something i need to talk to @RoaldBoerema about